README.mb

```diff
@@ -0,0 +1,17 @@
+# Welcome to Core Services exercise repository!
+
+If you got here by mistake and you are curious to know if we are hiring - visit the Outbrain careers page!
+https://careers.outbrain.com
+
+If you got here after your first interview, then all there is left is to wish you is good luck on your assignment!
+Change to the exersice you were instructed to perform, there you will find all the instructions you need
+
+## Exercises
+#### OutRain API
```

Comment:
open question - do we need this exercise at this point? I think consul-api does a good job to cover this as well

Reply:
I think each one checks different things, Outrain for example only checks code abilities while consul-api checks both system installation and code.
I think it will help us better fit an assignment to a candidate based on his knowledge